tests/unit/test_routes.py

```diff
@@ -495,171 +564,34 @@ def test_create_resource(module_client, module_db, fake_auth_from_oc, fake_algol
-
-
-def test_create_multiple_resources(module_client, module_db, fake_auth_from_oc, fake_algolia_save):
-    client = module_client
-    apikey = get_api_key(client)
-
-    # Happy Path
-    url1 = f"http://example.org/{str(datetime.now())}"
-    url2 = f"http://example.com/{str(datetime.now())}"
-    data = [dict(
-                name="Some Name",
-                url=url1,
-                category="New Category",
-                languages=["Python", "New Language"],
-                paid=False,
-                notes="Some notes"),
-            dict(
-                name="Some Other Name",
-                url=url2,
-                category="Different Category",
-                languages=["Python", "New Language", "JSON"],
-                paid=True,
-                notes="Some notes")
-            ]
-    response = client.post('/api/v1/resources',
-                json=data,
-                headers={'x-apikey': apikey})
-
-    assert (response.status_code == 200)
-    response_data = response.get_json().get("data")
-    assert (len(response_data) == 2)
-    for res in response_data:
-        assert (res.get("url") == url1 or res.get("url") == url2)
-
-    # Sad path (duplicate URL)
-    data = [dict(
-                name="Some Name",
-                url=url1,
-                category="New Category",
-                languages=["Python", "New Language"],
-                paid=False,
-                notes="Some notes")]
-    response = client.post('/api/v1/resources',
-                json=data,
-                headers={'x-apikey': apikey})
-    assert (response.status_code == 422)
-    assert (response.get_json().get("data") == None)
-    errors = response.get_json().get("errors")
-    assert (isinstance(errors, list))
-    assert (errors[0].get("index") == 0)
-    assert (errors[0].get("invalid-params").get("message"))
-    assert (errors[0].get("invalid-params").get("resource"))
-    assert (isinstance(errors[0].get("invalid-params").get("params"), list))
-
-    # Sad path, check index
-    data = [dict(
-                name="Some Name",
-                url="some.new.url",
-                category="New Category",
-                languages=["Python", "New Language"],
-                paid=False,
-                notes="Some notes"),
-            dict(
-                name="Other Name"
-            )]
-    response = client.post('/api/v1/resources',
-                json=data,
-                headers={'x-apikey': apikey})
+    response = client.post(
+        '/api/v1/resources',
+        json=[dict(notes="Missing Required fields")],
+        headers={'x-apikey': apikey}
+    )
     assert (response.status_code == 422)
-    assert (response.get_json().get("data") == None)
-    assert (response.get_json().get("errors")[0].get("index") == 1)
-
-
-def test_create_resource_wrong_type(module_client, module_db, fake_auth_from_oc, fake_algolia_save):
```

Comment:
I need to put my tests for the multiple resources change back in here, they were lost during the rebase